frontend/src/app/components/tableWithFilter/table.module.ts

```diff
@@ -14,7 +16,9 @@ import {TableComponent} from './table.component';
     TableModule,
     MultiSelectModule,
     FormsModule,
-    ProgressSpinnerModule
+    ProgressSpinnerModule,
+    ButtonModule,
+    RippleModule
   ],
```

Comment:
You have to import InputTextModule because you use it in the global filter textfield

Reply:
And InputNumberModule